ivy/functional/frontends/tensorflow/math.py

```diff
@@ -425,6 +425,14 @@ def nextafter(x1, x2, name=None):
     return ivy.nextafter(x1, x2)
 
 
+@with_unsupported_dtypes(
+    {"1.2.0": ("float16", "complex64", "complex128"), "1.8.0 and below": ("float16")},
+    "tensorflow",
+)
```

Comment:
Please include the uints as unsupported from 2.9.0 and below  in order to pass the test (i.e. `uint8`, `uint16`, ...)

Reply:
This commit was passed through the pre-commit hook